### ISSUE TITLE
Type mismatch: java.lang.Object is not a subtype of io.netty.util.concurrent.Future

### ISSUE THREAD

**Issue by dubinsky** [zio/zio-http#613]
ZIO: 1.0.12; zhttp 1.0.0.0-RC17 works fine; with 1.0.0.0-RC18 I get:

```
Fiber failed.
An unchecked error was produced.
java.lang.BootstrapMethodError: bootstrap method initialization exception
	at java.base/java.lang.invoke.BootstrapMethodInvoker.invoke(BootstrapMethodInvoker.java:194)
	at java.base/java.lang.invoke.CallSite.makeSite(CallSite.java:307)
	at java.base/java.lang.invoke.MethodHandleNatives.linkCallSiteImpl(MethodHandleNatives.java:258)
	at java.base/java.lang.invoke.MethodHandleNatives.linkCallSite(MethodHandleNatives.java:248)
	at zhttp.service.HttpRuntime.unsafeRun$$anonfun$4$$anonfun$2$$anonfun$2(HttpRuntime.scala:24)
	at zio.ZIO$.effectTotal$$anonfun$1(ZIO.scala:2819)
	at zio.internal.FiberContext.evaluateNow(FiberContext.scala:344)
	at zio.internal.FiberContext.evaluateLater$$anonfun$1(FiberContext.scala:776)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.invoke.LambdaConversionException: Type mismatch for instantiated parameter 0: class java.lang.Object is not a subtype of interface io.netty.util.concurrent.Future
	at java.base/java.lang.invoke.AbstractValidatingLambdaMetafactory.checkDescriptor(AbstractValidatingLambdaMetafactory.java:308)
	at java.base/java.lang.invoke.AbstractValidatingLambdaMetafactory.validateMetafactoryArgs(AbstractValidatingLambdaMetafactory.java:294)
	at java.base/java.lang.invoke.LambdaMetafactory.altMetafactory(LambdaMetafactory.java:503)
	at java.base/java.lang.invoke.BootstrapMethodInvoker.invoke(BootstrapMethodInvoker.java:138)
	... 10 more
```

What am I doing wrong?
Thanks!

**Comment by amitksingh1490** [comment]
@dubinsky. You are not doing anything wrong, we found that in RC18 there was an issue with HttpRuntime when used with scala3 and jdk 11+. It has been fixed in this PR #608. Please try the latest snapshot.

**Comment by dubinsky** [comment]
> ... It has been fixed in this PR #608. Please try the latest snapshot.

Yay! Seems to work with 1.0.0.0-RC18+8-fecadc20-SNAPSHOT!
Looking forward to a non-snapshot release ;)
Thanks!

**Comment by tusharmath** [comment]
Fixed in #608
